apps/nowcasting-app/components/helpers/globalState.tsx

```diff
@@ -1,6 +1,5 @@
 import { createGlobalState } from "react-hooks-global-state";
 import { AGGREGATION_LEVELS, getDeltaBucketKeys, SORT_BY, VIEWS } from "../../constant";
-import mapboxgl from "mapbox-gl";
```

Comment:
im guessing this wasnt used any more. Is there an automatic way to sweep unused imports? Perhaps for a different PR